crates/engine/tree/src/persistence.rs

```diff
@@ -236,7 +262,7 @@ impl PersistenceHandle {
 
     /// Tells the persistence task to remove blocks above a certain block number. The removed blocks
     /// are returned by the task.
```

Comment:
would be good to update the comment to reflect what the returned `B256` represents

Reply:
yep, makes sense, it should be the lowest block removed from the db, so the hash of block_num + 1